test/domain/contact_form/contact_form_dto_test.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:collaction_app/domain/contact_form/contact_form_dto.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  late ContactFormDto tContactForm;
+  setUp(() {
+    tContactForm =
+        ContactFormDto(email: 'test@test.com', message: 'Testing DTO');
```

Comment:
This is a Constant, can we push this to a fixture? `contact_form_fixture.dart`

Reply:
done.